native/android/src/main/java/com/nozbe/watermelondb/Database.kt

```diff
@@ -15,8 +15,21 @@ class Database(private val name: String, private val context: Context) {
                 if (name == ":memory:" || name.contains("mode=memory")) {
                     context.cacheDir.delete()
                     File(context.cacheDir, name).path
+                } else if (name.startsWith("/") || name.startsWith("file")) {
```

Comment:
I feel like in this case we should force user to add `*.db` extension to the file. 
Can you add a check for that and maybe some throw with verbose error so that it's clear what failed?

Reply:
done